packages/pay/src/open-payments.ts

```diff
@@ -110,10 +110,12 @@ export const fetchPaymentDetails = async (
   else if (destinationAccount) {
     const account = await queryAccount(destinationAccount)
     if (isAccount(account)) {
-      return createIncomingPayment(account, amountToDeliver)
+      return PaymentError.InvalidDestination
     } else {
       return account
```

Comment:
If this is an error, why are we supporting it in the input? Is there any case where we would need it provided?